collections/learn-to-code/index.md

```diff
@@ -5,12 +5,30 @@ items:
  - railsbridge/docs
  - freeCodeCamp/freeCodeCamp
  - leachim6/hello-world
+ - karan/Projects
+ - sindresorhus/awesome
+ - donnemartin/system-design-primer
+ - danistefanovic/build-your-own-x
+ - public-apis/public-apis
+ - EbookFoundation/free-programming-books
+ - amitness/learning
+ - mhinz/vim-galore
+ - tayllan/awesome-algorithms
+ - karlhorky/learn-to-program
+ - therebelrobot/awesome-workshopper
+ - jlevy/the-art-of-command-line
+ - papers-we-love/papers-we-love
+ - awesome-selfhosted/awesome-selfhosted
+ - ripienaar/free-for-dev
  - datasciencemasters/go
  - tuvtran/project-based-learning
  - zhiwehu/Python-programming-exercises
  - MunGell/awesome-for-beginners
  - appacademy/welcome-to-open
+ - Hacker0x01/hacker101
```

Comment:
This repo is the source code for a website. I haven't used hacker101.com, but I'm not sure the repo itself is very helpful as a "learn to code" resource.

Reply:
@bdresser Is this something you'd like to change, or is it ok as-is? If ok as-is I can go ahead and merge this since I'm on-call this week, but happy to wait if you'd like to make some changes!